frontend/src/scenes/data-warehouse/viewLinkLogic.tsx

```diff
@@ -8,7 +8,6 @@ import { databaseSceneLogic } from 'scenes/data-management/database/databaseScen
 import { loaders } from 'kea-loaders'
```

Comment:
Was there a blocking reason these changes were reverted?

REF: https://github.com/PostHog/posthog/pull/17287

Reply:
Ok yikes! None whatsoever. I think that was just a terrible merge on my part. Sorry about that Eric. I see it's back in now